docs/concepts/storage/volumes.md

```diff
@@ -81,6 +81,7 @@ Kubernetes supports several types of Volumes:
    * `hostPath`
    * `iscsi`
    * `local`
+   * `maprfs`
```

Comment:
maprfs is not a Kubernetes volume plugin, it's out of tree flex driver. Please do not include it in the list and Kubernetes docs. We do not list all possible flex drivers out there - they are just too many of them and Kubernetes does not ensure their quality.